h/static/styles/partials-v2/_base.scss

```diff
@@ -52,6 +52,11 @@ $max-phone-width: 500px;
 $tablet-width: 768px;
 $touch-target-size: 44px;
 
+// Misc
+// ----
+$search-result-container-margin-left: 30px;
+$search-result-container-margin-right: 60px;
```

Comment:
OK, let's just keep an eye on this in future and not let it become a dumping ground.